tests/FSharp.Core.UnitTests/FSharp.Core.UnitTests.fsproj

```diff
@@ -87,7 +87,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="**/*"  Exclude="**/*.bak;Directory.Build.Props;Directory.Build.targets" CopyToOutputDirectory="never" />
+    <Content Include="**/*" Exclude="**/*.bak;Directory.Build.Props;Directory.Build.targets;FSharp.Core.UnitTests.fsproj" CopyToOutputDirectory="never" />
```

Comment:
What's the rationale for introducing `FSharp.Core.UnitTests.fsproj`?

Reply:
Before:
![image](https://user-images.githubusercontent.com/8547855/69388809-68d5f180-0cca-11ea-9d7a-967e8d8c4bd7.png)
After:
![image](https://user-images.githubusercontent.com/8547855/69388831-7ee3b200-0cca-11ea-876b-e70eb5bd00a1.png)

I changed it, because I believed self reference of the fsproj was an unintended side effect of using  `<Content Include="**/*"` . I assumed that the preferred way for text editing the fsproj in VS is by choosing `Edit Project File` from context menu.

This is not connected with dotless floats, so feel free to ask for revert/to revert it.

Reply:
Looks good to me.